Homework Submission from Patrick Maulion

pmaulion opened the pull request.

pmaulion committed:
Basic Setup

Setting up HTML and CSS files

pmaulion committed:
H1 & NAV

Title and Nav tags

pmaulion committed:
Draft 1

Both Pages have been created w/ links/ text/ and pictures

pmaulion committed:
Rounded Pictures

Rounded “Assignment” picture borders, changed Bluejack photo

pmaulion committed:
CSS

Added new accents to styling of HR, incorporated “Cardinal” color
theme, wrote some notes in external CSS for better categorization,
added source to footer nav of TW wiki

pmaulion committed:
Changed Top Logo

Used Pixlr for editing stock TW Logo, removed H1 header

pmaulion committed:
Text Edit Changes

Proofread text paragraphs

pmaulion committed:
Moved images to Photos Folder

Moved images to Photos Folder for better organization

pmaulion committed:
added utf-8

rmjames commented on achievements.html:
```diff
@@ -0,0 +1,71 @@
+            |
+            <a href="https://en.wikipedia.org/wiki/Tiger_Woods" alt="Tiger Wiki">Source</a>
+        </nav>        
+    </footer>
```
@pmaulion the closing `</body` and `</html>` are missing 🔨

pmaulion replied:
Added closing body and html tags

rmjames commented on css/style.css:
```diff
@@ -0,0 +1,112 @@
+}
+
+.opacity {
+    opacity: .7%;
```
@pmaulion this is value from `0.0` - `1` 🔨

pmaulion replied:
Removed % on opacity section of CSS

rmjames commented on css/style.css:
```diff
@@ -0,0 +1,112 @@
+
+.opacity {
+    opacity: .7%;
+    border-color: #DDDDDD;
```
you could have used `rgba(221, 221, 221, 1.0)` or `hsla(0, 0%, 87%, 1.0)` to declare the color and opacity in one shot

rmjames commented on css/style.css:
```diff
@@ -0,0 +1,112 @@
+.opacity {
+    opacity: .7%;
+    border-color: #DDDDDD;
+    border-style: dotted;
```
when declaring borders remember to specify the `border-width`.  For example 
```border: 1px dotted hsla(0, 0%, 87%, 1.0)```

rmjames commented on css/style.css:
```diff
@@ -0,0 +1,112 @@
+#logo {
+    width: 1200px;
+    height: 300px;
+    align-content: center;
```
this declaration is used when the parent element has a display of flex (`display: flex`). We will learn about Flexbox later in the course

rmjames commented:
@pmaulion excellent work on this assignment, and good to see multiple commits.  Just make note of the corrections.

pmaulion committed:
Fixed changes on Assignment #1

Fixed opacity to 0.7 from 0.7%
Added html and body tags to achievements.html
Fixed indentation of achievements.html

rmjames commented:
👍
